### ISSUE TITLE
Changing PE directions between echoes (question)

### ISSUE THREAD

**Issue by BPaton** [CMRR-C2P/MB#259]
I am not in front of a scanner at the moment to check for myself but is it possible to choose different (opposite) phase encoding directions for the different echoes if using multi-echo e.g. Echo 1 PA, Echo 2 AP, Echo 3 PA etc.? Apologies if this is something obvious and I have missed how to do this.

Thanks.

**Comment by eauerbach** [comment]
No, this is not currently an option. It can look to see if it is possible to add.

**Comment by BPaton** [comment]
That would be great thanks. I was hoping to use SE MB for functional imaging in the spine and use the alternating PE direction echoes to look at per image ("real time") distortion corrections along the lines of this paper:

[https://doi.org/10.1016/j.neuroimage.2004.10.012](https://doi.org/10.1016/j.neuroimage.2004.10.012)
